system-tests/_support/index.js

```diff
@@ -245,15 +249,15 @@ export function createService(serviceName, serviceJsonPath) {
 }
 
 /**
- * Deletes a service using the dcos CLI
+ * Deletes a service from group TEST_UUID using the dcos CLI
  *
- * @param {String} serviceName - The service name which it will delete
+ * @param {String} serviceId - The service id which it will delete
  *
  */
-export function deleteService(serviceName) {
-  cy.exec(
-    `dcos marathon app remove /${Cypress.env("TEST_UUID")}/${serviceName}`
-  );
+export function deleteService(serviceId) {
+  cy.exec(`dcos marathon app remove ${serviceId}`);
```

Comment:
Same here. We are relying on `serviceId` being prefixed with `TEST_UUID` but it's possible that it wouldn't be and we should avoid logic like this.